handsontable/src/plugins/contextMenu/predefinedItems/alignment.js

```diff
@@ -4,6 +4,53 @@ import * as C from '../../../i18n/constants';
 
 export const KEY = 'alignment';
 
+/**
+ * @param  {string} htClassName The class name to check.
+ * @returns {boolean} Returns `true` if the cell has the provided class name.
+ */
+function getAlignmentComparator(htClassName) {
+  return function(row, col) {
+    const className = this.getCellMeta(row, col).className;
+
+    if (className && className.indexOf(htClassName) !== -1) {
+      return true;
+    }
+
+    return false;
+  };
+}
+
+/**
+ * @param {object} context The current context instance.
+ * @param {string} htClassName The class name to check.
+ * @returns {string} The value of aria-checked parameter.
+ */
+function ariaChecked(context, htClassName) {
```

Comment:
If possible, I'd not use the name `context` here. The name is too ambiguous. I guess the Handsontable instance is passed only here so I'd suggest naming it as `hot` or `hotInstance.`

Reply:
Fixed - context replaced with hot.